### ISSUE TITLE
Dynamic ASGs not processed properly when rule contains a comma-space list of ports

### ISSUE THREAD

**Issue by grenzr** [cloudfoundry/cf-networking-release#149]
## Issue

When a security group definition json contains a `ports` string with multiple ports separated by comma and space characters, only the port before the split is processed, meaning iptables rules on the diego-cells are incomplete.

Faulty:
```
name: cf_network
rules:
[
  {
  "protocol": "tcp",
  "destination": "10.156.161.0/25",
  "ports": "80, 443"
  }
]
```

Working:
```
name: cf_network
rules:
[
  {
  "protocol": "tcp",
  "destination": "10.156.161.0/25",
  "ports": "80,443"
  }
]
```

## Context

We upgraded from cf v19 to cf v20.2.0 and found that stratos UI was no longer able to contact the API. The CATs 'route_services' test also failed because it could not forward traffic to the app container. (see below for log)

## Steps to Reproduce

Create a security group per above, 

## Expected result

iptables rules are created on the diego-cells to match the security group config, with rules covering both 80 and 443.

## Current result

iptables rules only exist for port 80

## Possible Fix

We're guessing something is being split by a comma somewhere, but haven't been able to identify where as yet.

## Additional Context

CATS failure output:

```
[Fail] [route_services] Route Services when a route binds to a service when service broker returns a route service url [It] a request to the app is routed through the route service
/go/src/github.com/cloudfoundry/cf-acceptance-tests/route_services/route_services.go:89

2022-05-16T10:53:14.78+0000 [APP/PROC/WEB/0] OUT 2022/05/16 10:53:14 Request Body:
2022-05-16T10:53:14.78+0000 [APP/PROC/WEB/0] OUT 2022/05/16 10:53:14 Forwarding to: https://CATS-6-APP-bc246de71ce0bdfc.app.xxx/
2022-05-16T10:53:17.83+0000 [APP/PROC/WEB/0] OUT 2022/05/16 10:53:17 http: proxy error: dial tcp 10.156.147.77:443: connect: connection refused
```

**Comment by vlast3k** [comment]
We also have not been able to find the root cause yet.
So far it looks like disabling dynamic AGS via the ops file is mitigating the issue

**Comment by geofffranks** [comment]
@vlast3k @grenzr Thanks for reporting this. Looks related to not pulling in the [TrimSpace logic we used for c2c policies](https://github.com/cloudfoundry/silk-release/blob/develop/src/code.cloudfoundry.org/vxlan-policy-agent/planner/planner_linux.go#L309-L310) into the [new security_group_rule converter](https://github.com/cloudfoundry/silk-release/blob/develop/src/code.cloudfoundry.org/cni-wrapper-plugin/netrules/security_group_rule.go#L53).

**Comment by vlast3k** [comment]
i've prepared the pull request to fix this.
Took me a bit more time to validate it as i unintentionally chose a somewhat "broken" landscape.
Diego cells had ~200k iptables rules with ~30k rules each assigned to several apps. Enabling Dynamic ASGs, caused the Diego Cell Drain to hang for hours, deleting ~1 rule/second.  Also the silk-daemon was waiting for "/var/vcap/data/container-metadata/store.json" to become empty, but for some reason it was not emptied and it was just waiting 10 minutes.
After i purged the Policy Server DB, everything went fine

**Comment by geofffranks** [comment]
thankjs @vlast3k - yeah having more than 100k iptables rule per cell is going to cause problems in general. i'll reach out on slack about this specifically.

**Comment by geofffranks** [comment]
The fix has been released in silk-release [v3.8.0](https://github.com/cloudfoundry/silk-release)!
